src/dfx/src/main.rs

```diff
@@ -180,7 +180,9 @@ fn print_error_and_diagnosis(err: Error, error_diagnosis: Diagnosis) {
 fn main() {
     let cli_opts = CliOpts::parse();
     let (verbose_level, log) = setup_logging(&cli_opts);
-    let identity = cli_opts.identity;
+    let identity = cli_opts
+        .identity
+        .or_else(|| std::env::var("DFX_IDENTITY").ok());
```

Comment:
clap has a feature for this, see https://github.com/dfinity/sdk/pull/2934/files#diff-525ab7c425ed736af9f5d76feaa51616197f59fee5db465772ee27c534a57eccR15

Reply:
Nice.  Does that mean that the env var is documented automatically?  Yes it does.  Cool.